packages/amplify-app/src/project-configs/project-config-android.json

```diff
@@ -1,11 +0,0 @@
-{
-  "projectName": "amplifyStarter",
-  "version": "2.0",
-  "frontend": "android",
-  "android": {
-    "config": {
-      "ResDir": "app/src/main/res"
```

Comment:
Where do we store this info now?

Reply:
it's now in amplify-cli (the android stuff has always been there, but now ios and javascript is there as well)